radiation/fortran/radlw/Makefile

```diff
@@ -34,7 +34,7 @@ SRC = \
   radiation_clouds.F \
   radiation_aerosols.F \
   radsw_main.F \
-  radlw_main.F  \
+  radlw_main2.F  \
```

Comment:
more of the main.F, vs main2.F -- are you sure you want this change committed?

Reply:
Sorry, I should have cleaned this up more, I'll work on it

Reply:
Ok radlw_main2.F is the version with `lwrad` broken out into smaller loops, which is what I want here